src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -5621,12 +5622,11 @@ def stacks(self):
         self._load_stacks()
         return self._stacks
 
-    def get_flex_raw_function_app_stacks(self, cmd, location, runtime):
-        stacks_api_url = '/providers/Microsoft.Web/locations/{}/functionAppStacks?' \
-                         'api-version=2020-10-01&removeHiddenStacks=true&removeDeprecatedStacks=true&stack={}'
+    def get_raw_function_app_stacks(self, cmd, location, runtime, sku):
+        stacks_api_url = '/providers/Microsoft.Web/locations/{}/functionAppStacks?api-version=2023-01-01&removeHiddenStacks=true&removeDeprecatedStacks=true&stack={}&sku={}'  # pylint: disable=line-too-long
```

Comment:
Instead of passing "FC1" from every calling function, why don't you just make it so that this url is

 stacks_api_url = '/providers/Microsoft.Web/locations/{}/functionAppStacks?api-version=2023-01-01&removeHiddenStacks=true&removeDeprecatedStacks=true&stack={}&**sku=FC1**'

This way, we don't have to update all existing functions. I think we should only accept a sku param to the RuntimeHelper class and methods if this class is used for stuff other than flex and this sku value can vary based on those applications. But my understanding is that all of this is for flex and we would have to use the same sku value right?

Reply:
Made the class as generic in terms of SKU because the stacks API is not Flex specific anymore.